test/wpt/status/WebCryptoAPI.json

```diff
@@ -4,13 +4,5 @@
   },
   "historical.any.js": {
     "skip": "Not relevant in Node.js context"
-  },
-  "idlharness.https.any.js": {
```

Comment:
This change is completely unrelated to existing 2 commits. Please either open a different pull request or commit this change in a new one.

Reply:
This cannot be separated since the update to interfaces covers all interfaces. To accomodate that notice that I have 2 separate commits in this PR and https://github.com/nodejs/node/labels/commit-queue-rebase, one commit clearly just updates the interfaces, the other updates `URL.parse` and resolves the failing wpt expectation.

Reply:
This updates webcrypto api. I didn't understand how it is related to interfaces or url parse?

Reply:
`git node wpt interfaces` updates all WebIDL interfaces, `URL`, `WebCryptoAPI`, everything else that's defined with WebIDLs.